zorch/commit/testing/ajtai_test.py

```
@@ -0,0 +1,190 @@
+        message = self._message()
+        r1 = _ternary_witness(self.ring, self.rng, _COLS)
+        r2 = _ternary_witness(self.ring, self.rng, _COLS)
+        c1 = self.scheme.commit(self.b0, self.b1, message, r1)
+        c2 = self.scheme.commit(self.b0, self.b1, message, r2)
+        same = all(
+            bool(
+                np.array_equal(
+                    np.asarray(x).astype(object), np.asarray(y).astype(object)
+                )
+            )
+            for x, y in zip(c1.t0.limbs, c2.t0.limbs)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)